src/console2.sol

```diff
@@ -1543,4 +1590,1373 @@ library console2 {
         _sendLogPayload(abi.encodeWithSignature("log(address,address,address,address)", p0, p1, p2, p3));
     }
 
+    // Log Functions with Color functionality.
```

Comment:
Were these codegen'd or did you do it manually by copying all the existing log methods and just e.g. multi-cursor editing them? Just curious

Reply:
I copied the existing log functions and modified them through a regex